src/i18n/cs_CZ/buildings.ts

```diff
@@ -585,19 +663,19 @@ export default {
     21: {
         caption: 'Pyrotechnická služba PČR',
         color: '#663300',
-        coins: 50,
-        credits: 5_000_000,
+        coins: 35,
+        credits: 200_000,
         levelPrices: {
-            credits: [10_000, 50_000, ...Array(18).fill(100_000)],
-            coins: [10, 15, ...Array(18).fill(20)],
+            credits: [10_000, 50_000, ...Array(3).fill(100_000)],
```

Comment:
This building has only 3 levels?

Reply:
that's right. it only has 5 in total, the first costs 10k, the second 50k and then it goes on 3x with 100k :)